pyanaconda/installation.py

```diff
@@ -331,6 +331,9 @@ def run_pre_install():
 
         if can_install_bootloader:
             payload.requirements.add_packages(storage.bootloader.packages, reason="bootloader")
+        if flags.cmdline.get("fips") == "1":
```

Comment:
To make this work you have to change the code to this.
```suggestion
        if flags.flags.cmdline.getbool("fips"):
```

Sorry for the ``flags.flags`` that is just a wrong import but we will fix that in the future later.